### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by NagaSatyaGeeth**
https://etl4gy.csb.app/
image

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]

File: style.css [hunk 1]
```diff
@@ -0,0 +1,234 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@100;200;300;400;500;600;700;800&display=swap");
+:root {
+	font-family: "Montserrat", sans-serif;
+	padding: 0px;
+	margin: 0px;
+	font-size: 62.5%;
+	box-sizing: border-box;
+	--primary-color: #21b0f2;
+	--header-bg-light-color: #fff;
+	--header-boxshadow-light-color: rgba(0, 0, 0, 0.3);
```
Comment (Divya70) [inline style.css:10]: use variable for color

File: style.css [hunk 2]
```diff
@@ -0,0 +1,234 @@
...
+	justify-content: center;
+	align-items: center;
+}
+
+.btn {
+	font-size: 2rem;
+	font-weight: 600;
+	padding: 1rem;
+	background-color: var(--primary-color);
+	color: #fff;
```
Comment (Divya70) [inline style.css:213]: here too

File: style.css [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+	border: none;
+	margin: 4rem 0rem 0rem 10rem;
```
Comment (Divya70) [inline style.css:215]: why this much margin 10 rem ?

File: style.css [hunk 4]
```diff
@@ -0,0 +1,234 @@
...
+}
+
+.btn-2 {
+	font-size: 1.5rem;
+}
+
+a {
+	text-decoration: none;
+	color: #fff;
```
Comment (Divya70) [inline style.css:224]: use color variable

**Review by Divya70 — APPROVED** [review]
suggested some chnages rest is good

**Pull request merged**